pinot-core/src/main/java/com/linkedin/pinot/core/io/readerwriter/impl/FixedByteSingleColumnSingleValueReaderWriter.java

```diff
@@ -163,9 +163,8 @@ private int getBufferId(int row) {
   }
 
   private void addBuffer() {
-    LOGGER.info("Allocating {} bytes for column {} for segment {}", _chunkSizeInBytes, _columnName,
-        _memoryManager.getSegmentName());
-    PinotDataBuffer buffer = _memoryManager.allocate(_chunkSizeInBytes, _columnName);
+    LOGGER.info("Allocating {} bytes for segment:column {}", _chunkSizeInBytes, _allocationContext);
```

Comment:
Since this class is expected to be for some generic use, perhaps you can just remove the "segment:column" from the log message.

Reply:
Done.